addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -117,6 +129,9 @@ channel-type.netatmo.lastStatusStore.description = Zeigt den Zeitpunkt an, an de
 channel-type.netatmo.timeUtc.label = Zeitpunkt Messung
```

Comment:
Just a reminder for myself: I maybe want to change this label to "Letzte Messung".